src/wellKnownDidConfiguration/wellKnownDidConfiguration.test.ts

```diff
@@ -40,13 +40,10 @@ describe('Well Known Did Configuration integration test', () => {
   beforeAll(async () => {
     const address = await startContainer()
     await connect(address)
-  }, 20_000)
-
-  beforeAll(async () => {
```

Comment:
the split in the two beforeAlls meant that we had 20 seconds allowed to starting the container, and then 30 seconds allotted to the setup. now we have 30 seconds for everything.

Reply:
Do you think I should revert this change?

Reply:
I think so, it had its reason